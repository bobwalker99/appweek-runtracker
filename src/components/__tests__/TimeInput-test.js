/**
 */
 'use strict';

const React = require('React');
const ReactTestRenderer = require('react-test-renderer');
const Text = require('Text');

import TimeInput from '../TimeInput.react';

describe('TimeInput', () => {
  it('renders correctly', () => {
    const instance = ReactTestRenderer.create(
      <TimeInput style={{}} />
    );
    expect(instance.toJSON()).toMatchSnapshot();
  });
});
