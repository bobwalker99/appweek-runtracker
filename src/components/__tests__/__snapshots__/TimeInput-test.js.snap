// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TimeInput renders correctly 1`] = `
<View>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
  >
    I'm a time input
  </Text>
</View>
`;
